src/Footer.jsx

```diff
@@ -0,0 +1,239 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import locales from 'mwp-config/locales';
+
+import Bounds from './layout/Bounds';
+import Chunk from './layout/Chunk';
+import Flex from './layout/Flex';
+import FlexItem from './layout/FlexItem';
+import InlineBlockList from './layout/InlineBlockList';
+import GridList from './layout/GridList';
+import Icon from './media/Icon';
+import Section from './layout/Section';
+import SelectInput from './forms/SelectInput';
+import Stripe from './layout/Stripe';
+import withMatchMedia from './utils/components/withMatchMedia';
+
+import { getSocialLinks } from './utils/getSocialLinks';
+
+export const FooterCategory = (props) => (
+	<Chunk>
+		<h4 className="text--bold margin--bottom">{props.header}</h4>
+		<GridList
+			columns={{
+				default: 2,
+				medium: 1,
+			}}
+			className="text--secondary text--small"
+			itemClassNames="footer-item"
+			items={props.items}
+		/>
+	</Chunk>
+);
+
+
+export const SocialIconsList = (language) => {
```

Comment:
I'd change the arg `language` to `locale` to make it clear that the function is expecting a locale code

Reply:
It actually takes a language abbreviation (`en`), not a locale code (`en-US`). It _could_ be changed to take a locale code though

Reply:
my concern here is that `getSocialLinks` expects a locale

Reply:
You're right. I was mixing this up for the way `AppBadges` works in mup-web 😳